### ISSUE TITLE
question: create a new conversation / issue internally

### ISSUE THREAD

**Issue by danielzzz** [chatwoot/chatwoot#3984]
Hi, I was wondering if it's possible to create a conversation / issue from within chatwoot.
The use case for this would be to manage issues detected internally, that don't involve external users.

thanks, dan

**Comment by ChrisvanChip** [comment]
It would be an interesting feature, it would really improve the team's productivity (as less software is necessary) and we don't have to use issue tracking software anymore.

**Comment by sojan-official** [comment]
As a temporary workaround, you could create contacts with some sort of an internal format for email address 
contact1@acme.local -> create an outbound conversation for this contact.  and keep track of the rest of the conversation over this thread.

**Comment by danielzzz** [comment]
yes, this was my thinking for now. but I haven't find a way to create a new outbound conversation in chatwoot. could you please point me to some docs on how to do it, pls?

thanks, dan

**Comment by ChrisvanChip** [comment]
In your contact list (on your sidebar), you can select a contact and press the blue chat icon.

![Screenshot_20220216-121639_Chrome](https://user-images.githubusercontent.com/31969757/154253951-de4092ce-36ce-4887-95ca-631af90bb4f2.jpg)

**Comment by stonecharioteer** [comment]
This issue is being closed due to lack of activity for over a year. [For more details, visit this discussion.](https://github.com/orgs/chatwoot/discussions/12810)
